src/py42/clients/cases.py

```diff
@@ -0,0 +1,175 @@
+from enum import Enum
+
+from py42.util import parse_timestamp_to_milliseconds_precision
+
+
+class CaseStatus(Enum):
+    OPEN = u"OPEN"
+    CLOSED = u"CLOSED"
+
+
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documentation <https://developer.code42.com/api/#tag/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of methods for managing file events associated with a given case.
+
+        Returns :
+            :class:`py42.services.casesfileevents.FileEventsService`
+        """
+        return self._file_events
+
+    def create(
+        self, name, subject=None, assignee=None, description=None, findings=None
+    ):
+        """Creates a new case.
+
+        Args:
+            name (str): Name of the case.
+            subject (str, optional): User UID of a subject of a case.
+            assignee (str, optional): User UID of the assignee.
+            description (str, optional): Description of the case
+            findings (str, optional): Observations of the case.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        return self._cases_service.create(
+            name,
+            subject=subject,
+            assignee=assignee,
+            description=description,
+            findings=findings,
+        )
+
+    @staticmethod
+    def _make_range(begin_time, end_time):
+        if not begin_time or not end_time:
```

Comment:
I am wondering if it would make sense to have `end_time` default to the current time if not provided, so that way it is easy to get all cases from a point back in time until now. Does that make sense?

Reply:
I would agree with this approach, you could similarly use the epoch time if the user only supplied an end time and no begin time.

Reply:
Done.